insights/specs/default.py

```diff
@@ -948,6 +948,46 @@ def is_mod_loaded_for_ss(broker):
     sssd_config = simple_file("/etc/sssd/sssd.conf")
     subscription_manager_id = simple_command("/usr/sbin/subscription-manager identity")  # use "/usr/sbin" here, BZ#1690529
     subscription_manager_installed_product_ids = simple_command("/usr/bin/find /etc/pki/product-default/ /etc/pki/product/ -name '*pem' -exec rct cat-cert --no-content '{}' \;")
+
+    @datasource(Ps, HostContext)
+    def proc_swap_memory(broker):
+        ps = broker[Ps]
+        pids = [p for p in sorted(os_listdir("/proc/")) if p.isdigit()]
```

Comment:
Better get the `PID` from the Ps combiner directly, so you do not need to list the whole `"/proc"` dir and iterate the Ps again to get the "COMMAND".

Reply:
Ack, updated the code